This patch keeps PosgreSQL's configure script from removing the rpath from
Perl's linker options, because otherwise the PL/Perl module can't find
libperl.so (bsc#578053).

--- config/perl.m4.orig
+++ config/perl.m4
@@ -100,9 +100,7 @@ if test "$PORTNAME" = "win32" ; then
 		fi
 	fi
 else
-	pgac_tmp1=`$PERL -MExtUtils::Embed -e ldopts`
-	pgac_tmp2=`$PERL -MConfig -e 'print $Config{ccdlflags}'`
-	perl_embed_ldflags=`echo X"$pgac_tmp1" | sed -e "s/^X//" -e "s%$pgac_tmp2%%" -e ["s/ -arch [-a-zA-Z0-9_]*//g"]`
+	perl_embed_ldflags=`$PERL -MExtUtils::Embed -e ldopts`
 fi
 AC_SUBST(perl_embed_ldflags)dnl
 if test -z "$perl_embed_ldflags" ; then
--- configure.orig
+++ configure
@@ -7537,9 +7537,7 @@ if test "$PORTNAME" = "win32" ; then
 		fi
 	fi
 else
-	pgac_tmp1=`$PERL -MExtUtils::Embed -e ldopts`
-	pgac_tmp2=`$PERL -MConfig -e 'print $Config{ccdlflags}'`
-	perl_embed_ldflags=`echo X"$pgac_tmp1" | sed -e "s/^X//" -e "s%$pgac_tmp2%%" -e "s/ -arch [-a-zA-Z0-9_]*//g"`
+	perl_embed_ldflags=`$PERL -MExtUtils::Embed -e ldopts`
 fi
 if test -z "$perl_embed_ldflags" ; then
 	{ $as_echo "$as_me:${as_lineno-$LINENO}: result: no" >&5
